src/app/services/sms/__tests__/sms.service.spec.ts

```diff
@@ -80,180 +67,6 @@ describe('sms.service', () => {
   afterEach(async () => await dbHandler.clearDatabase())
   afterAll(async () => await dbHandler.closeDatabase())
 
-  describe('sendFormDeactivatedSms', () => {
```

Comment:
why need to remove?

Reply:
No longer used. For admins, we're switch them directly over to Postman.